gammapy/estimators/tests/test_flux_map.py

```diff
@@ -233,6 +240,21 @@ def test_flux_map_properties(wcs_flux_map, reference_model):
     assert_allclose(fluxmap.sqrt_ts.data, 1)
     assert_allclose(fluxmap.ts.data[:, 0, 0], [0, 3])
 
+    assert_allclose(fluxmap.success.data[:, 0, 1], [False, True])
+    assert_allclose(fluxmap.flux.data[:, 0, 1], [np.nan, 9e-13])
+    assert_allclose(fluxmap.flux_err.data[:, 0, 1], [np.nan, 9e-14])
+
+    assert_allclose(fluxmap.eflux.data[:, 0, 1], [np.nan, 2.30258509e-12])
+    assert_allclose(fluxmap.e2dnde_err.data[:, 0, 1], [np.nan, 1e-13])
+
+
+def test_flux_map_failed_properties(wcs_flux_map, reference_model):
+    fluxmap = FluxMaps(wcs_flux_map, reference_model)
+    fluxmap.filter_success_nan = False
+
```

Comment:
I would suggest to add an explicit test for the getter/setter mechanism here, e.g.
```
assert fluxmap.filter_success_nan == False
```

Reply:
done